phpunit.xml.dist

```diff
@@ -1,13 +1,14 @@
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" colors="true"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
The installed version of PHPUnit is controlled via Composer configuration. So, please use the installed file otherwise we have to change URL / version also here.